src/components/Person/Person.jsx

```diff
@@ -1 +1,50 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Person.module.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partnerStatus = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className={styles.Person}>
+      <h2 className={styles.Person__name}>
```

Comment:
![image](https://github.com/mate-academy/react_person/assets/46566190/a1aca844-5c5e-4af7-bf71-0a5fb872bcb1)
- Tests are trying to find elements with another classnames